packages/segmented-control/src/SegmentedControlOption.styles.ts

```diff
@@ -0,0 +1,211 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { Mode, fontFamilies, focusRing } from '@leafygreen-ui/tokens';
+import { Size } from './types';
+
+/**
+ * Styles
+ */
+export const optionThemeStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    --base-text-color: ${palette.gray.dark1};
+    --base-background-color: rgba(255, 255, 255, 0);
+    --base-shadow-color: rgba(255, 255, 255, 0);
+    // Hover
+    --hover-text-color: ${palette.gray.dark3};
+    // Selected
+    --active-text-color: ${palette.white};
+    // Disabled
+    --disabled-text-color: ${palette.gray.light1};
+    // Divider
+    --divider-background-color: ${palette.gray.light1};
+  `,
+  [Theme.Dark]: css`
+    --base-text-color: ${palette.gray.light1};
+    --base-background-color: rgba(255, 255, 255, 0);
+    --base-shadow-color: rgba(255, 255, 255, 0);
+    // Hover
+    --hover-text-color: ${palette.gray.light2};
+    // Selected
+    --active-text-color: ${palette.black};
+    // Disabled
+    --disabled-text-color: ${palette.gray.dark2};
+    // Divider
+    --divider-background-color: ${palette.gray.dark2};
+  `,
+};
+
+export const optionSizeStyle: Record<Size, string> = {
+  [Size.Small]: css`
+    --font-size: 12px;
+    --line-height: 16px;
+    --padding-inline: 12px;
+    --text-transform: uppercase;
+    --font-weight: 700;
+    --divider-height: 12px;
+  `,
+  [Size.Default]: css`
+    --font-size: 13px;
+    --line-height: 24px;
+    --padding-inline: 12px; // left/right
+    --text-transform: none;
+    --font-weight: 500;
+    --divider-height: 18px;
+  `,
+  [Size.Large]: css`
+    --font-size: 16px;
+    --line-height: 28px;
+    --padding-inline: 12px;
+    --text-transform: none;
+    --font-weight: 500;
+    --divider-height: 20px;
+  `,
+};
+
+export const optionPaddingStyle: Record<Theme, Record<Size, string>> = {
+  [Theme.Light]: {
+    [Size.Small]: css`
+      --padding-block: 3px;
+    `,
+    [Size.Default]: css`
+      --padding-block: 3px;
+    `,
+    [Size.Large]: css`
+      --padding-block: 4px;
+    `,
+  },
+  [Theme.Dark]: {
+    [Size.Small]: css`
+      --padding-block: 3px;
+    `,
+    [Size.Default]: css`
+      --padding-block: 4px;
+    `,
+    [Size.Large]: css`
+      --padding-block: 5px;
+    `,
+  },
+};
```

Comment:
I think these should be the same now in both modes as well

Reply:
The paddings are different for each mode

Reply:
updated!